### ISSUE TITLE
Allow to use 'sqrt()' without syntax error and quotation marks.

### ISSUE THREAD

**Issue by kamilmichna** [kamilmichna/calc#1]
For example, calc -c "sqrt(2)" works perfectly, but calc -c sqrt(2) ithrows a syntax error. It`s propably a problem with commander.

**Comment by cdrani** [comment]
No, the problem is not commanderjs. The `(` and `)` are special characters that need to be escaped in the shell. Enclosing them in single/double quotes or escaping them `calc -c sqrt\(4\)` is the solution. I would just change the code to allow something like this `calc -c sqrt4`.

**Comment by kamilmichna** [comment]
@cdrani  thank you very much for response mate!
